src/components/idleMembers/card/index.tsx

```diff
@@ -1,13 +1,13 @@
-import { FC } from 'react';
+import { FC, useState } from 'react';
 import classNames from '@/components/idleMembers/card/card.module.scss';
 
 type Props = {
   idleMemberUserName: string
 }
 
 const Card: FC<Props> = ({ idleMemberUserName }) => {
-  const imageGenerator = (name: string) => `${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}/${name}/img.png`;
-
+  const [assigneeProfilePic, setAssigneeProfilePic] = useState(`${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}/${idleMemberUserName}/img.png`);
```

Comment:
Any specific reason to use state management here ? 
check this out :point_down: 
**https://github.com/Real-Dev-Squad/website-members/blob/develop/src/components/member-card/index.js**

Reply:
I am not sure why the original editor used state management😅, my task was only to make small changes in this PR  https://github.com/Real-Dev-Squad/website-status/pull/156

Reply:
Got it. @whyDontI can't we avoid using state management here ?

Reply:
Yes. We can directly have idlememberURL or if an error then we can have dummyProfileURL. I think we can avoid useState.